apps/mtnclock/settings.js

```diff
@@ -0,0 +1,29 @@
+(function(back) {
+  var STORAGE = require('Storage')
+
+  var FILE = "mtnclock.json";
+
+  // Load settings
+  var SETTINGS  = Object.assign({
+    // default values
+    showWidgets: false,
+  }, STORAGE.readJSON(FILE, true) || {});
+
+  function writeSettings() {
+    STORAGE.writeJSON(FILE, SETTINGS);
+  }
+
+  // Show the menu
+  E.showMenu({
+    "" : { "title" : "Mountain Clock" },
+    "< Back" : () => back(),
+    'Show widgets': {
+      value: !!SETTINGS.showWidgets,  // !! converts undefined to false
+      format: value => value ? "On" : "Off",
```

Comment:
For booleans we can leave this out and it'll default to a standard display :) (for example, [like in this commit](https://github.com/espruino/BangleApps/commit/42c2b5c8a2)'s settings file changes)

```suggestion
```